generators/client/templates/react/package.json.ejs

```diff
@@ -92,35 +91,35 @@ limitations under the License.
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
     <%_ } _%>
-    "copy-webpack-plugin": "5.0.0",
-    "core-js": "2.6.5",
+    "copy-webpack-plugin": "5.0.2",
+    "core-js": "3.0.0",
```

Comment:
while updating dependencies, I think we should also incorporate app wise changes, if any suggested. This would help to ensure generated code is up to date

Reply:
@vishal423 I agree but I really do not have the time to deal with this right now. TBH I am doing the updates in hope to get the React tests on CI more stable as they are kind of flaky but no luck so far 😞 .